rdfunit-junit/src/main/java/org/aksw/rdfunit/junit/RLOGStatement.java

```diff
@@ -0,0 +1,42 @@
+package org.aksw.rdfunit.junit;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.hp.hpl.jena.rdf.model.ResourceFactory;
+import org.aksw.rdfunit.tests.results.RLOGTestCaseResult;
+import org.aksw.rdfunit.tests.results.TestCaseResult;
+import org.junit.runners.model.Statement;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class RLOGStatement extends Statement {
+
+    private final RdfUnitJunitStatusTestExecutor rdfUnitJunitStatusTestExecutor;
+    private final RdfUnitJunitTestCase child;
+
+    RLOGStatement(RdfUnitJunitStatusTestExecutor rdfUnitJunitStatusTestExecutor, RdfUnitJunitTestCase child) {
+        this.rdfUnitJunitStatusTestExecutor = rdfUnitJunitStatusTestExecutor;
+        this.child = child;
```

Comment:
Improve Naming: child (?) referred to the Child-Test but here the context is different and it's simply the thing referencing the test.